src/frontend/query_commands.ml

```diff
@@ -293,15 +306,13 @@ let dispatch pipeline (type a) : a Query_protocol.t -> a =
           match text, print with
           | Stack_or_heap_enclosing.No_alloc { reason }, true ->
               ret (`String ("not an allocation (" ^ reason ^ ")"))
-          | Stack_or_heap_enclosing.Alloc_mode _, true ->
-              (* TODO: decide correct behavior and implement *)
-              (* let str =
-                match alloc_mode |> Mode.Alloc.locality |> Mode.Locality.check_const with
+          | Stack_or_heap_enclosing.Alloc_mode alloc_mode, true ->
+              let str =
+                match alloc_mode |> Mode.Alloc.proj (Comonadic Areality) |> Mode.Locality.Guts.check_const_conservative with
```

Comment:
I think it's a little cleaner to bind this large computation to a variable instead of just tucking it into the `match ... with`.